recipes/tomviz/yum_requirements.txt

```diff
@@ -0,0 +1,22 @@
+mesa-libGL-devel
+mesa-libGLU-devel
+mesa-libEGL-devel
+libX11-devel
+libXt-devel
+libXtst-devel
+libXrender-devel
+libXext-devel
+libXdmcp-devel
+libXcomposite-devel
+libXcursor-devel
+libxcb-devel
+libXau-devel
+libXi-devel
+libXScrnSaver-devel
+libXrandr-devel
+alsa-lib-devel
+xcb-util-devel
+xorg-x11-server-Xvfb
```

Comment:
Should it be duplicated here too? It's currently in `meta.yaml`:
```yaml
     - {{ cdt('xorg-x11-server-xvfb') }}  # [linux]
```
I think it's true for most of the packages in this file. Looks like it's a mixture of system packages with the conda-provided ones.

Reply:
Not sure its actually needed, it a carry over from paraview. I am going to try removing it.

Reply:
The others need to be in the `yum_requirements.txt`, not sure if they need to be duplicated in `meta.yml` but that seem to be what other recipes do.